docs/sources/enterprise/usage-insights.md

```diff
@@ -55,3 +55,17 @@ It shows two kinds of information:
 In the search view, you can sort dashboards using these insights data. It helps you find unused or broken dashboards or discover most viewed ones.
 
 {{< docs-imagebox img="/img/docs/enterprise/improved_search.png" max-width="650px" class="docs-image--no-shadow" >}}
+
+## Data source insights
+
+Data source insights gives you information about how much a data source has been used and how many errors it has returned. It shows the last thirty days of data.
```

Comment:
Data source insights give you information about how much of the data source has been used and how many errors it has returned. It shows the last thirty days of data.